### PR TITLE
write info about properties in user

### PR DESCRIPTION
**Pull request by Slashek**

### CODE REVIEW

File: marketplace_builder/views/pages/tutorials/users/create-user-property.liquid [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* [Codebase](/how-platformos-works/platformos-workflow)
+* [Users](/tutorials/users/users)
+
+## Steps
+
+Creating a property for all Users is a one-step process:
+
+<div data-autosteps></div>
+
+### Step 1: Create yml file
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/users/create-user-property.liquid]: I got lost somewhere between step 1 and step 1 ;)

Comment (Slashek) [reply marketplace_builder/views/pages/tutorials/users/create-user-property.liquid]: try again starting from the beginning :P

**Comment by pavelloz** [comment]
btw rebase _nav-links - its nav-links now :)

**Review by diana-lakatos — APPROVED** [review]
Reviewed and edited. Thank you!

**Comment by diana-lakatos** [comment]
Closes #463

**Pull request merged**